packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -1,30 +1,140 @@
 # Dropdown
 
+## Basic usage
+
+```html
+<div class="dropdown">
+  <button
+    data-toggle="dropdown"
+    data-target="#DropdownDemo1"
+  >
+    toggle
+  </button>
+  <div class="Dropdown is-open" id="DropdownDemo1">
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#info" />
+      </svg>
+      <span>Information</span>
+    </a>
+    </a>
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#link" />
+      </svg>
+      <span>More links</span>
+    </a>
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#profile" />
+      </svg>
+      <span>Profile</span>
+    </a>
+    <a href="#" class="d-flex">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#help" />
+      </svg>
+      <span>Help</span>
+    </a>
+  </div>
+</div>
+```
+
+## Advanced usage
+
 ```html
-<div class="Dropdown is-open">
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#info" />
-    </svg>
-    <span>Information</span>
-  </a>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#link" />
-    </svg>
-    <span>More links</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#profile" />
-    </svg>
-    <span>Profile</span>
-  </a>
-  <a href="#" class="d-flex">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#help" />
-    </svg>
-    <span>Help</span>
+<div class="dropdown">
+  <button
+    data-toggle="dropdown"
+    data-target="#DropdownDemo2"
+    data-reference="parent"
+    data-placement="top-start"
+    data-offset="5"
```

Comment:
Could we have values from the design here as well? People will copy these examples and they should get it correct right away.

IIRC, there is either 4 or 8 since 5 is not in [our spacing scale](https://spirit.supernova-docs.io/spirit/latest/tokens/measure.html).